### ISSUE TITLE
Canu Failed with exit code 133. (rc=34048)

### ISSUE THREAD

**Issue by Munnj90** [marbl/canu#427]
Hello,

I'm currently trying to run Canu using MinION data.  I'm using base called data (Fasta) from DeepNano.

I used the following command and got the error code listed above:

```
dominions-MacBook-Air-2:bin *$ canu -d ~/Desktop/*/MinION220317/ -p S_gold_McHugh_220317_EDIT genomeSize=9M gnuplotTested=true minReadLength=100 minOverlapLength=50 -nanopore-raw ~/Desktop/*/MinION220317/S_gold_McHugh_220317_EDIT.fasta
--   Reason: image not found
-- Detected Java(TM) Runtime Environment '1.8.0_121' (from 'java').
-- Detected 4 CPUs and 8 gigabytes of memory.
-- No grid engine detected, grid disabled.
--
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    8 GB memory for stage 'bogart (unitigger)'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    6 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    6 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    6 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    2 GB memory for stage 'read error detection (overlap error adjustment)'.
-- Allowed to run   4 jobs concurrently, and use up to   1 compute thread  and    1 GB memory for stage 'overlap error adjustment'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    8 GB memory for stage 'utgcns (consensus)'.
-- Allowed to run   4 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlap store parallel bucketizer'.
-- Allowed to run   4 jobs concurrently, and use up to   1 compute thread  and    8 GB memory for stage 'overlap store parallel sorting'.
-- Allowed to run   4 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    8 GB memory for stage 'meryl (k-mer counting)'.
-- Allowed to run   2 jobs concurrently, and use up to   2 compute threads and    8 GB memory for stage 'falcon_sense (read correction)'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    6 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    6 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   4 compute threads and    6 GB memory for stage 'minimap (overlapper)'.
--
-- This is canu parallel iteration #1, out of a maximum of 2 attempts.
--
-- Final error rates before starting pipeline:
--   
--   genomeSize          -- 9000000
--   errorRate           -- 0.048
--   
--   corOvlErrorRate     -- 0.144
--   obtOvlErrorRate     -- 0.144
--   utgOvlErrorRate     -- 0.144
--   
--   obtErrorRate        -- 0.144
--   
--   cnsErrorRate        -- 0.144
--
--
-- BEGIN CORRECTION
--
----------------------------------------
-- Starting command on Tue Mar 28 14:23:28 2017 with 309.338 GB free disk space

    /usr/local/Cellar/canu/1.4/Darwin-amd64/bin/gatekeeperCreate \
      -minlength 100 \
      -o /Users/*/Desktop/*/MinION220317//correction/S_gold_McHugh_220317_EDIT.gkpStore.BUILDING \
      /Users/*/Desktop/*/MinION220317//correction/S_gold_McHugh_220317_EDIT.gkpStore.gkp \
    > /Users/*/Desktop/*/MinION220317//correction/S_gold_McHugh_220317_EDIT.gkpStore.BUILDING.err 2>&1
sh: line 4: 93261 Trace/BPT trap: 5       /usr/local/Cellar/canu/1.4/Darwin-amd64/bin/gatekeeperCreate -minlength 100 -o /Users/*/Desktop/*/MinION220317//correction/S_gold_McHugh_220317_EDIT.gkpStore.BUILDING /Users/*/Desktop/*/MinION220317//correction/S_gold_McHugh_220317_EDIT.gkpStore.gkp > /Users/*/Desktop/*/MinION220317//correction/S_gold_McHugh_220317_EDIT.gkpStore.BUILDING.err 2>&1

-- Finished on Tue Mar 28 14:23:28 2017 (lickety-split) with 309.338 GB free disk space
----------------------------------------
ERROR:
ERROR:  Failed with exit code 133.  (rc=34048)
ERROR:
================================================================================
Don't panic, but a mostly harmless error occurred and canu failed.

canu failed with 'gatekeeper store exists, but contains no reads'.
```

Any help would be greatly appreciated!

**Comment by skoren** [comment]
Have you tried any of the test datasets to make sure your Canu installation is working? Is this Canu compiled from source or a downloaded binary? What is in the ```MinION220317//correction/S_gold_McHugh_220317_EDIT.gkpStore.BUILDING.err``` file?

**Comment by Munnj90** [comment]
Hi Skoren,

Thanks for getting back to me.  I believe the issue was that it was incorrectly installed (I believe is used a Binary (homebrew installed)).  I then used the source version and got it running.

I've however reached another issue.

I apologise in advance...I'm not entirely sure I know what I'm doing...any help again is appreciated and thank you for responding so quickly!

```
perl ~/Downloads/canu/Darwin-amd64/bin/canu -d ~/Desktop/*/MinION220317_2/ -p *_220317_EDIT genomeSize=9M gnuplotTested=true minReadLength=100 minOverlapLength=50 -nanopore-raw ~/Desktop/*/MinION220317/*_220317_EDIT.fasta
-- Canu v1.4 (+82 commits) r8077 145ff3c4fa3479417e312794dbeb77e4f4cca940.
-- Detected Java(TM) Runtime Environment '1.8.0_121' (from 'java').
--
-- Generating assembly '*_220317_EDIT' in '/Users/*/Desktop/*/MinION220317_2'
--
-- Detected 4 CPUs and 8 gigabytes of memory.
-- No grid engine detected, grid disabled.
--
-- Run   1 job  concurrently using    8 GB and   4 CPUs for stage 'meryl'.
-- Run   1 job  concurrently using    6 GB and   4 CPUs for stage 'mhap (cor)'.
-- Run   1 job  concurrently using    8 GB and   4 CPUs for stage 'overlapper (obt)'.
-- Run   1 job  concurrently using    8 GB and   4 CPUs for stage 'overlapper (utg)'.
-- Run   2 jobs concurrently using    8 GB and   2 CPUs for stage 'falcon_sense'.
-- Run   4 jobs concurrently using    2 GB and   1 CPU  for stage 'ovStore bucketizer'.
-- Run   4 jobs concurrently using    8 GB and   1 CPU  for stage 'ovStore sorting'.
-- Run   1 job  concurrently using    2 GB and   4 CPUs for stage 'read error detection'.
-- Run   4 jobs concurrently using    1 GB and   1 CPU  for stage 'overlap error adjustment'.
-- Run   1 job  concurrently using    8 GB and   4 CPUs for stage 'bogart'.
-- Run   1 job  concurrently using    8 GB and   4 CPUs for stage 'consensus'.
--
-- Parameters:
--
--  genomeSize        9000000
--
--  Overlap Generation Limits:
--    corOvlErrorRate 0.1440 ( 14.40%)
--    obtOvlErrorRate 0.1440 ( 14.40%)
--    utgOvlErrorRate 0.1440 ( 14.40%)
--
--  Overlap Processing Limits:
--    corErrorRate    0.5000 ( 50.00%)
--    obtErrorRate    0.1440 ( 14.40%)
--    utgErrorRate    0.1440 ( 14.40%)
--    cnsErrorRate    0.1440 ( 14.40%)
--
-- This is canu parallel iteration #1, out of a maximum of 2 attempts.
--
--
-- BEGIN CORRECTION
--
----------------------------------------
-- Starting command on Tue Mar 28 16:09:42 2017 with 307.881 GB free disk space

    cd correction
    /Users/*/Downloads/canu/Darwin-amd64/bin/gatekeeperCreate \
      -minlength 100 \
      -o ./*_220317_EDIT.gkpStore.BUILDING \
      ./*_220317_EDIT.gkpStore.gkp \
    > ./*_220317_EDIT.gkpStore.BUILDING.err 2>&1

-- Finished on Tue Mar 28 16:09:43 2017 (1 second) with 307.856 GB free disk space
----------------------------------------
--
-- WARNING: gnuplot failed; no plots will appear in HTML output.
--
----------------------------------------
--
-- In gatekeeper store 'correction/*_220317_EDIT.gkpStore':
--   Found 29181 reads.
--   Found 52917506 bases (5.87 times coverage).
--
--   Read length histogram (one '*' equals 251.47 reads):
--        0    999  17603 **********************************************************************
--     1000   1999   3295 *************
--     2000   2999   2220 ********
--     3000   3999   1758 ******
--     4000   4999   1395 *****
--     5000   5999    993 ***
--     6000   6999    711 **
--     7000   7999    429 *
--     8000   8999    274 *
--     9000   9999    148 
--    10000  10999     98 
--    11000  11999     53 
--    12000  12999     40 
--    13000  13999     21 
--    14000  14999     11 
--    15000  15999     10 
--    16000  16999     10 
--    17000  17999      9 
--    18000  18999     12 
--    19000  19999     12 
--    20000  20999      8 
--    21000  21999      5 
--    22000  22999      8 
--    23000  23999      5 
--    24000  24999      6 
--    25000  25999      0 
--    26000  26999      2 
--    27000  27999      3 
--    28000  28999      1 
--    29000  29999      5 
--    30000  30999      3 
--    31000  31999      1 
--    32000  32999      0 
--    33000  33999      2 
--    34000  34999      0 
--    35000  35999      0 
--    36000  36999      0 
--    37000  37999      0 
--    38000  38999      2 
--    39000  39999      1 
--    40000  40999      2 
--    41000  41999      1 
--    42000  42999      0 
--    43000  43999      2 
--    44000  44999      2 
--    45000  45999      1 
--    46000  46999      0 
--    47000  47999      0 
--    48000  48999      0 
--    49000  49999      0 
--    50000  50999      0 
--    51000  51999      1 
--    52000  52999      0 
--    53000  53999      0 
--    54000  54999      1 
--    55000  55999      0 
--    56000  56999      1 
--    57000  57999      1 
--    58000  58999      1 
--    59000  59999      1 
--    60000  60999      0 
--    61000  61999      2 
--    62000  62999      1 
--    63000  63999      1 
--    64000  64999      0 
--    65000  65999      0 
--    66000  66999      0 
--    67000  67999      0 
--    68000  68999      1 
--    69000  69999      1 
--    70000  70999      2 
--    71000  71999      0 
--    72000  72999      0 
--    73000  73999      0 
--    74000  74999      0 
--    75000  75999      0 
--    76000  76999      1 
--    77000  77999      0 
--    78000  78999      0 
--    79000  79999      0 
--    80000  80999      1 
--    81000  81999      0 
--    82000  82999      0 
--    83000  83999      0 
--    84000  84999      0 
--    85000  85999      0 
--    86000  86999      0 
--    87000  87999      0 
--    88000  88999      0 
--    89000  89999      0 
--    90000  90999      0 
--    91000  91999      0 
--    92000  92999      0 
--    93000  93999      0 
--    94000  94999      0 
--    95000  95999      0 
--    96000  96999      0 
--    97000  97999      0 
--    98000  98999      0 
--    99000  99999      0 
--   100000 100999      0 
--   101000 101999      1 
--   102000 102999      1 
--   103000 103999      0 
--   104000 104999      0 
--   105000 105999      0 
--   106000 106999      0 
--   107000 107999      0 
--   108000 108999      0 
--   109000 109999      0 
--   110000 110999      0 
--   111000 111999      0 
--   112000 112999      0 
--   113000 113999      0 
--   114000 114999      0 
--   115000 115999      0 
--   116000 116999      0 
--   117000 117999      0 
--   118000 118999      0 
--   119000 119999      0 
--   120000 120999      0 
--   121000 121999      1 
-- Meryl attempt 1 begins.
----------------------------------------
-- Starting concurrent execution on Tue Mar 28 16:09:43 2017 with 307.855 GB free disk space (1 processes; 1 concurrently)

    cd correction/0-mercounts
    ./meryl.sh 1 > ./meryl.000001.out 2>&1

-- Finished on Tue Mar 28 16:10:01 2017 (18 seconds) with 307.668 GB free disk space
----------------------------------------
-- Meryl finished successfully.
----------------------------------------
-- Starting command on Tue Mar 28 16:10:01 2017 with 307.668 GB free disk space

    cd correction/0-mercounts
    /Users/*/Downloads/canu/Darwin-amd64/bin/meryl \
      -Dh \
      -s ./*_220317_EDIT.ms16 \
    > ./*_220317_EDIT.ms16.histogram \
    2> ./*_220317_EDIT.ms16.histogram.info

-- Finished on Tue Mar 28 16:10:01 2017 (lickety-split) with 307.668 GB free disk space
----------------------------------------
--
-- WARNING: gnuplot failed; no plots will appear in HTML output.
--
----------------------------------------
--
-- WARNING: gnuplot failed; no plots will appear in HTML output.
--
----------------------------------------
-- For mhap overlapping, set repeat k-mer threshold to 524.
--
-- Found 52479791 16-mers; 47694194 distinct and 44290458 unique.  Largest count 151056.
--
-- OVERLAPPER (mhap) (correction)
--
-- Set corMhapSensitivity=high based on read coverage of 5.
--
-- PARAMETERS: hashes=768, minMatches=2, threshold=0.73
--
-- Given 6 GB, can fit 9000 reads per block.
-- For 5 blocks, set stride to 2 blocks.
-- Logging partitioning to 'correction/1-overlapper/partitioning.log'.
-- Configured 4 mhap precompute jobs.
-- Configured 5 mhap overlap jobs.
-- mhap precompute attempt 1 begins with 0 finished, and 4 to compute.
----------------------------------------
-- Starting concurrent execution on Tue Mar 28 16:10:02 2017 with 307.855 GB free disk space (4 processes; 1 concurrently)

    cd correction/1-overlapper
    ./precompute.sh 1 > ./precompute.000001.out 2>&1
    ./precompute.sh 2 > ./precompute.000002.out 2>&1
    ./precompute.sh 3 > ./precompute.000003.out 2>&1
    ./precompute.sh 4 > ./precompute.000004.out 2>&1

-- Finished on Tue Mar 28 16:19:16 2017 (554 seconds) with 307.343 GB free disk space
----------------------------------------
-- All 4 mhap precompute jobs finished successfully.
-- mhap attempt 1 begins with 0 finished, and 5 to compute.
----------------------------------------
-- Starting concurrent execution on Tue Mar 28 16:19:16 2017 with 307.343 GB free disk space (5 processes; 1 concurrently)

    cd correction/1-overlapper
    ./mhap.sh 1 > ./mhap.000001.out 2>&1
    ./mhap.sh 2 > ./mhap.000002.out 2>&1
    ./mhap.sh 3 > ./mhap.000003.out 2>&1
    ./mhap.sh 4 > ./mhap.000004.out 2>&1
    ./mhap.sh 5 > ./mhap.000005.out 2>&1

-- Finished on Tue Mar 28 16:21:13 2017 (117 seconds) with 307.341 GB free disk space
----------------------------------------
--
-- 4 mhap jobs failed:
--   job 1-overlapper/results/000001.ovb FAILED.
--   job 1-overlapper/results/000002.ovb FAILED.
--   job 1-overlapper/results/000003.ovb FAILED.
--   job 1-overlapper/results/000004.ovb FAILED.
--
-- mhap attempt 2 begins with 1 finished, and 4 to compute.
----------------------------------------
-- Starting concurrent execution on Tue Mar 28 16:21:13 2017 with 307.341 GB free disk space (4 processes; 1 concurrently)

    cd correction/1-overlapper
    ./mhap.sh 1 > ./mhap.000001.out 2>&1
    ./mhap.sh 2 > ./mhap.000002.out 2>&1
    ./mhap.sh 3 > ./mhap.000003.out 2>&1
    ./mhap.sh 4 > ./mhap.000004.out 2>&1

-- Finished on Tue Mar 28 16:21:13 2017 (lickety-split) with 307.341 GB free disk space
----------------------------------------
--
-- 4 mhap jobs failed:
--   job 1-overlapper/results/000001.ovb FAILED.
--   job 1-overlapper/results/000002.ovb FAILED.
--   job 1-overlapper/results/000003.ovb FAILED.
--   job 1-overlapper/results/000004.ovb FAILED.
--
================================================================================
Don't panic, but a mostly harmless error occurred and canu failed.

canu failed with 'failed to compute mhap overlaps.  Made 2 attempts, jobs still failed'.
```

**Comment by skoren** [comment]
This is most likely an issue with the java install on your system, what's in the mhap.00000[1-4].out files and precompute. 00000[1-4]?

Before you spend too much time on this assembly, it looks like you only have 6X coverage, we recommend at least 20x so you're unlikely to get much assembly with so little data.

**Comment by Munnj90** [comment]
I don't seem to have those files (as far as i can tell...), should they be in the output directory (e.g. "-d ~/Desktop/*/MinION220317_2/")?

Ok, thank you very much for your help! :)

**Comment by skoren** [comment]
They would be under ```/MinION220317_2/correction/1-overlapper```

**Comment by Munnj90** [comment]
Sorry my bad, i looked in the wrong folder.

Please see below:


mhap.000001.out

```
Running job 1 based on command line options.
Running block 000001 in query 000001
INVALID OVERLAP     8954 (len   1735)     4932 (len   1864) hangs   2023    228 -     88   1768 flip 1

```
mhap.000002.out

```
Running job 2 based on command line options.
Running block 000001 in query 000002
INVALID OVERLAP    26905 (len    193)      967 (len    366) hangs    584    152 -    103    256 flip 1
```

mhap.000003.out

```
Running job 3 based on command line options.
Running block 000002 in query 000003
INVALID OVERLAP    17917 (len    260)    14863 (len    410) hangs   1749    173 -     60    344 flip 0

```
mhap.000004.out

```
Running job 4 based on command line options.
Running block 000003 in query 000004
INVALID OVERLAP    26881 (len    340)    24034 (len   3479) hangs    326   4481 -   2351    400 flip 0
```

mhap.000005.out

```
Running job 5 based on command line options.
Running block 000004 in query 000005
Running with these settings:
--filter-threshold = 5.0E-6
--help = false
--max-shift = 0.2
--min-olap-length = 116
--min-store-length = 0
--no-rc = false
--no-self = false
--no-tf = false
--num-hashes = 768
--num-min-matches = 2
--num-threads = 4
--ordered-kmer-size = 12
--ordered-sketch-size = 1536
--repeat-idf-scale = 10.0
--repeat-weight = 0.9
--settings = 0
--store-full-id = false
--supress-noise = 0
--threshold = 0.73
--version = false
-f = ../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz
-h = false
-k = 16
-p = 
-q = queries/000005
-s = ./blocks/000004.dat

Reading in filter file ../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz.
Time (s) to read filter file: 0.11824443500000001
Read in k-mer filter for sizes: [16]
Processing files for storage in reverse index...
Stored 4302 sequences in the index.
Processed 4302 unique sequences (fwd and rev).
Time (s) to read and hash from file: 2.0694313120000003
Time (s) to score and output to self: 0.8687887350000001
Total scoring time (s): 0.869947859
Total time (s): 2.9394889
MinHash search time (s): 3.055631297
Total matches found: 396
Average number of matches per lookup: 0.18410041841004185
Average number of table elements processed per lookup: 774.187819618782
Average number of table elements processed per match: 4205.247474747474
Average % of hashed sequences hit per lookup: 0.09129417928283494
Average % of hashed sequences hit that are matches: 4.6875
Average % of hashed sequences fully compared that are matches: 67.9245283018868
```

precompute.000001.out

```
Running job 1 based on command line options.
Dumping reads from 1 to 9000 (inclusive).
Starting mhap precompute.
Running with these settings:
--filter-threshold = 5.0E-6
--help = false
--max-shift = 0.2
--min-olap-length = 116
--min-store-length = 0
--no-rc = false
--no-self = false
--no-tf = false
--num-hashes = 768
--num-min-matches = 2
--num-threads = 4
--ordered-kmer-size = 12
--ordered-sketch-size = 1536
--repeat-idf-scale = 10.0
--repeat-weight = 0.9
--settings = 0
--store-full-id = false
--supress-noise = 0
--threshold = 0.73
--version = false
-f = ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz
-h = false
-k = 16
-p = ./000001.input.fasta
-q = .
-s = 

Reading in filter file ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz.
Time (s) to read filter file: 0.246468808
Read in k-mer filter for sizes: [16]
Processing FASTA files for binary compression...
Current # sequences loaded and processed from file: 5000...
Current # sequences loaded and processed from file: 10000...
Current # sequences loaded and processed from file: 15000...
Processed 17782 sequences (fwd and rev).
Read, hashed, and stored file ./000001.input.fasta to ./000001.input.dat.
Time (s): 177.687651552
Total time (s): 177.687957426
```

precompute.000002.out

```
Running job 2 based on command line options.
Dumping reads from 9001 to 18000 (inclusive).
Starting mhap precompute.
Running with these settings:
--filter-threshold = 5.0E-6
--help = false
--max-shift = 0.2
--min-olap-length = 116
--min-store-length = 0
--no-rc = false
--no-self = false
--no-tf = false
--num-hashes = 768
--num-min-matches = 2
--num-threads = 4
--ordered-kmer-size = 12
--ordered-sketch-size = 1536
--repeat-idf-scale = 10.0
--repeat-weight = 0.9
--settings = 0
--store-full-id = false
--supress-noise = 0
--threshold = 0.73
--version = false
-f = ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz
-h = false
-k = 16
-p = ./000002.input.fasta
-q = .
-s = 

Reading in filter file ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz.
Time (s) to read filter file: 0.105872274
Read in k-mer filter for sizes: [16]
Processing FASTA files for binary compression...
Current # sequences loaded and processed from file: 5000...
Current # sequences loaded and processed from file: 10000...
Current # sequences loaded and processed from file: 15000...
Processed 17770 sequences (fwd and rev).
Read, hashed, and stored file ./000002.input.fasta to ./000002.input.dat.
Time (s): 175.659808265
Total time (s): 175.65995876600002
```

precompute.000003.out

```
Running job 3 based on command line options.
Dumping reads from 18001 to 27000 (inclusive).
Starting mhap precompute.
Running with these settings:
--filter-threshold = 5.0E-6
--help = false
--max-shift = 0.2
--min-olap-length = 116
--min-store-length = 0
--no-rc = false
--no-self = false
--no-tf = false
--num-hashes = 768
--num-min-matches = 2
--num-threads = 4
--ordered-kmer-size = 12
--ordered-sketch-size = 1536
--repeat-idf-scale = 10.0
--repeat-weight = 0.9
--settings = 0
--store-full-id = false
--supress-noise = 0
--threshold = 0.73
--version = false
-f = ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz
-h = false
-k = 16
-p = ./000003.input.fasta
-q = .
-s = 

Reading in filter file ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz.
Time (s) to read filter file: 0.119100564
Read in k-mer filter for sizes: [16]
Processing FASTA files for binary compression...
Current # sequences loaded and processed from file: 5000...
Current # sequences loaded and processed from file: 10000...
Current # sequences loaded and processed from file: 15000...
Processed 17724 sequences (fwd and rev).
Read, hashed, and stored file ./000003.input.fasta to ./000003.input.dat.
Time (s): 158.301965712
Total time (s): 158.302224757
```

precompute.000004.out

```
Running job 4 based on command line options.
Dumping reads from 27001 to 29181 (inclusive).
Starting mhap precompute.
Running with these settings:
--filter-threshold = 5.0E-6
--help = false
--max-shift = 0.2
--min-olap-length = 116
--min-store-length = 0
--no-rc = false
--no-self = false
--no-tf = false
--num-hashes = 768
--num-min-matches = 2
--num-threads = 4
--ordered-kmer-size = 12
--ordered-sketch-size = 1536
--repeat-idf-scale = 10.0
--repeat-weight = 0.9
--settings = 0
--store-full-id = false
--supress-noise = 0
--threshold = 0.73
--version = false
-f = ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz
-h = false
-k = 16
-p = ./000004.input.fasta
-q = .
-s = 

Reading in filter file ../../0-mercounts/S_gold_McHugh_220317_EDIT.ms16.frequentMers.ignore.gz.
Time (s) to read filter file: 0.115894622
Read in k-mer filter for sizes: [16]
Processing FASTA files for binary compression...
Processed 4302 sequences (fwd and rev).
Read, hashed, and stored file ./000004.input.fasta to ./000004.input.dat.
Time (s): 38.851569996
Total time (s): 38.851818032000004
```

**Comment by skoren** [comment]
Ah I think this is a bug with using very short overlap/read lengths. Your setting is ```minReadLength=100 minOverlapLength=50```. Try increasing both to be at least 150bp and re-run from scratch in a new folder.

**Comment by Munnj90** [comment]
I've done so and it doesn't look great...see below contig.fasta file

Do you think it is an issue of not enough data/reads/coverage or are my settings causing an issue?

Thanks again!

```
>tig00000001 len=13426 reads=39 covStat=-26.18 gappedBases=no class=contig suggestRepeat=no suggestCircular=no
TTTTTTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTGTTTTGTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTGTTTTTTTTTTATTTTTTTTTTTTTTTTTTTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTATTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTGTTTTTTTTTTTTTTTTT
TGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTAATTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTCTCCCCCCGCTCCCCGTCCCGCC
GGGCCGACCCGCCGACCAGGCGGCGTTGCGGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTCTCC
CCATGGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTCGTTGTCGCCCCCGGCAA
CACCCACCGTAGTGGCTCGGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTCTGTCCCCCGGCGCGATCACCCTGT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTCCCGCCACCGGTTCCCGGTCCCC
CTGTCATTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
TTTTTTTTTTTTTTTTTTTTTTTTTA

```

**Comment by skoren** [comment]
Yes, the run finished fine, you probably have some more sequence in the unassembled.fasta file but yes, 5x is too low for assembly, I expect you have <3x after correction.
